lib/core/Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Wojciech Danilo <wojciech.danilo@gmail.com>"]
 edition = "2018"
 
 [lib]
-crate-type = ["cdylib"]
+crate-type = ["rlib", "cdylib"]
```

Comment:
Why we need `rlib`?

Reply:
We need it to make the crate available in the tests.

See https://github.com/rust-lang/cargo/issues/6659